DepositApi.UnitTests/ControllersTests/CalculateDepositApiControllerUnitTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DepositApi.UnitTests.Utility;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using DepositApi.Controllers;
+using DepositApi.Core.Models;
+using Moq;
+
+namespace DepositApi.UnitTests.ControllersTests
+{
+    [TestFixture]
+    class CalculateDepositApiControllerUnitTests
+    {
+        [Test]
+        public async Task GetAsync_ProperMethodCall()
```

Comment:
Generally: create more informative name for tests. For example `GetAsync_ExpectPercentCalculationAsyncMethodCalled`